### PR TITLE
:bug: Palba bugfixing 01

### PR DESCRIPTION
**Pull request by pabloalba**
Fixes:

https://tree.taiga.io/project/penpot/issue/9435
https://tree.taiga.io/project/penpot/issue/9350

### CODE REVIEW

File: frontend/src/app/main/ui/components/context_menu_a11y.scss [hunk 1]
```diff
@@ -35,8 +36,12 @@
     border: $s-2 solid var(--panel-border-color);
     background-color: var(--menu-background-color);
     overflow: auto;
+
     & .separator {
-      height: $s-12;
+      border-top: solid 1px var(--color-background-quaternary);
```
Comment (superalex) [inline frontend/src/app/main/ui/components/context_menu_a11y.scss]: We don't use "px" units, I think this should be a "$s-1", you can check all the available values in frontend/resources/styles/common/refactor/spacing.scss

File: frontend/src/app/main/ui/components/context_menu_a11y.scss [hunk 2]
```diff
@@ -35,8 +36,12 @@
...
+      border-right: 0;
```
Comment (superalex) [inline frontend/src/app/main/ui/components/context_menu_a11y.scss]: Are those 0 values for right, left and bottom really needed?

File: frontend/src/app/main/ui/components/context_menu_a11y.scss [hunk 3]
```diff
@@ -45,6 +50,7 @@
 
     .context-menu-item {
       display: flex;
+
```
Comment (superalex) [inline frontend/src/app/main/ui/components/context_menu_a11y.scss:50]: Why adding those extra lines in the commit?

File: frontend/src/app/main/ui/dashboard/sidebar.scss [hunk 4]
```diff
@@ -370,6 +378,10 @@
 
 .profile-separator {
   height: $s-6;
+  border-top: solid 1px var(--color-background-quaternary);
```
Comment (superalex) [inline frontend/src/app/main/ui/dashboard/sidebar.scss]: Same here

**Comment by belen-albeza** [comment]
Hi, it looks like the history panel bug could benefit from a Playwright test for future regressions? I also saw this bug in Taiga https://tree.taiga.io/project/penpot/issue/9492 , which I don't know if it's also fixed by this (in this case, we could add that interaction to the test too)

**Comment by pabloalba** [comment]
> Hi, it looks like the history panel bug could benefit from a Playwright test for future regressions?

ok


> I also saw this bug in Taiga https://tree.taiga.io/project/penpot/issue/9492 , which I don't know if it's also fixed by this (in this case, we could add that interaction to the test too)

No, that is a different issue

**Pull request merged**